codeforboston/vaccinatema
# Fully combine home and search pages!

This PR:
* Adds the search bar to the home page
* Hooks it up to the map / list view
* Shows only available sites before first search
* Deletes the search page

### Test Plan:
* Load the website.
* See that only available sites are displayed on the map and list view.
* Enter a search, ensure the results on the map and list view change accordingly!

<img width="1140" alt="Screen Shot 2021-03-05 at 7 10 29 PM" src="https://user-images.githubusercontent.com/8495791/110187702-4b596800-7de7-11eb-9d94-ed348a018ffa.png">
<img width="1136" alt="Screen Shot 2021-03-05 at 7 10 44 PM" src="https://user-images.githubusercontent.com/8495791/110187703-4c8a9500-7de7-11eb-860c-60311f3e3a78.png">
